frontend/packages/knative-plugin/src/catalog/event-source-data.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  EventSourceApiServerModel,
+  EventSourceContainerModel,
+  EventSourceKafkaModel,
+  EventSourcePingModel,
+  EventSourceSinkBindingModel,
+} from '../models';
+
+const eventSourceCatalogProviderData: {
+  [key: string]: { documentationUrl?: string; description?: string; provider?: string };
+} = {
+  [EventSourceApiServerModel.kind]: {
+    documentationUrl:
+      'https://docs.openshift.com/container-platform/4.6/serverless/event_sources/serverless-apiserversource.html',
```

Comment:
am not sure if we should add documentation link , until CRD provides it as it's not consistent some places it OCP docs and some it's upstream and even version if fixed here i.r 4.6

Reply:
As discussed, removed documentation URL.